worlds/ahit/test/TestActs.py

```diff
@@ -0,0 +1,35 @@
+from worlds.ahit.Regions import act_chapters
+from worlds.ahit.Rules import act_connections
+from . import HatInTimeTestBase
+
+
+class TestActs(HatInTimeTestBase):
+    def run_default_tests(self) -> bool:
+        return False
+      
+    def testAllStateCanReachEverything(self):
+        pass
+
+    options = {
+        "ActRandomizer": 2,
+        "EnableDLC1": 1,
+        "EnableDLC2": 1,
+        "ShuffleActContracts": 0,
+    }
+
+    def test_act_shuffle(self):
+        for i in range(1000):
```

Comment:
why?

Reply:
It's rolling 1000 seeds in an attempt to find any act shuffle layouts where a region ends up inaccessible. There's probably a better way to do this though.

Reply:
Changed it to 300 since 1000 is probably not necessary.

Reply:
instead of calling `world.setup()` it'd be better if it only called the relevant code you're trying to test as well, since you don't need to generate a full multiworld just to check entrances. probably with different seeds that you can log to debug.